### PR TITLE
Added delete endpoint for temp shipment

### PR DESCRIPTION
**Pull request by jbolles94**

### CODE REVIEW

File: src/tempShipment/controller.js [hunk 1]
```diff
@@ -33,3 +35,19 @@ async function createOne(req, res) {
     "creating temp shipment"
   );
 }
+
+/**
+ * Create a new temp shipment given a manifest
```
Comment (joshplumley) [inline src/tempShipment/controller.js:40]: Delete a new ...

File: src/tempShipment/controller.js [hunk 2]
```diff
@@ -33,3 +35,19 @@ async function createOne(req, res) {
...
+ */
+async function deleteOne(req, res) {
+  ExpressHandler(
+    async () => {
+      const { tsid } = req.params;
+
+      // delete temp shipment
+      await TempShipment.deleteOne({ _id: tsid });
```
Comment (joshplumley) [inline src/tempShipment/controller.js:48]: should we throw an error is this tsid doesnt exist?

File: src/tempShipment/controller.js (hunk 1 above)
Comment (jbolles94) [reply src/tempShipment/controller.js]: Removed.

File: src/tempShipment/controller.js (hunk 2 above)
Comment (jbolles94) [reply src/tempShipment/controller.js:53]: Resolved

**Review by joshplumley — APPROVED** [review]

**Pull request merged**